core/templates/pages/contact-page/contact-page.component.html

```diff
@@ -122,36 +122,29 @@ <h2 [innerHTML]="'I18N_CONTACT_PAGE_OTHER_INQUIRIES_HEADING' | translate"></h2>
   }
 
   :host {
-    .oppia-contact-page-content button {
+    .oppia-contact-page-content a.oppia-primary-link-button {
+      border-radius: 5px;
+      color: white;
+      display: inline-flex;
       font-size: 100%;
       font-weight: bold;
       height: 40px;
       width: 250px;
     }
 
     @media (max-width: 1010px) and (min-width: 843px) {
-      .oppia-contact-page-content button {
+      .oppia-contact-page-content a.oppia-primary-link-button {
```

Comment:
This sort of reaches "into" the primary button class. Can we instead pass different custom classes into the button element depending on the screen size? Otherwise this violates encapsulation somewhat, and is susceptible to breaking if the internals of the button component implementation are changed.

Similarly for other files -- if we want to treat the button as a component then ideally all configuration is done through the interface of the component (i.e. attributes).

Reply:
Makes sense, done